featurebyte/models/historical_feature_table.py

```diff
@@ -22,7 +29,7 @@ class HistoricalFeatureTableModel(BaseFeatureOrTargetTableModel):
     # Id of the feature list used to compute the historical feature table. None if the feature list
     # was not saved.
     feature_list_id: Optional[PydanticObjectId]
-    num_features: Optional[int] = Field(default=None)
+    features_info: Optional[List[FeatureInfo]] = Field(default=None)
```

Comment:
Would it be useful to provide methods / properties like `feature_names` and `feature_ids` to access the feature name and feature ids more conveniently?